atlas_admin/templates/instance_summary.html

```diff
@@ -10,6 +10,10 @@
         <a href="{{ envVars['baseURL'] }}/{{ instance.instance.path }}"
             target="_blank">{{ envVars['baseURL'] }}/{{ instance.instance.path }}</a>
     </div>
+    <div class="columns twelve">
```

Comment:
Remove this, I think it is more likely to cause confusion, the vast majority of the time this link will 404

Reply:
I'd rather not. This is helpful when you are trying to go look at all the sites that are having a similar problem. Nor do the links 404.

Reply:
The reason that they don't 404 from your local is because you have a copy of the production database (which is uncommon). For 99% of sites on dev and test, there is not an analogous site on production so the link would 404. On prod, this is a duplicate of the link above it.

If you want it for debugging, keep it in your local version but I don't think it belongs in the PR

Reply:
got it